delegation/src/Pos/Delegation/Logic/VAR.hs

```diff
@@ -439,10 +439,10 @@ dlgApplyBlocks dlgBlunds = do
                 SomeBatchOp $ map GS.DelPostedThisEpoch $
                 HS.toList duPrevEpochPosted
         pure $ edgeOp <> transCorrections <> postedOp
-    applyBlock ((Right block), _) = do
+    applyBlock ((ComponentBlockMain header block), _) = do
```

Comment:
Can you rename `block` to `payload`, please (because it's actually payload, not whole block)?